main.js

```diff
@@ -55,16 +62,52 @@ function removeColor(button, clickedClass){
 }
 
 function startActivity() {
-  preventEmptyInput();
-  //prevent empty input
-  //prevent e in numbers
+  event.preventDefault();
+  preventButtons();
+  preventAccomplish();
+  preventMinutes();
+  preventSeconds();
   //if no errors, create new instance
-  //switch to timer page
+  // hideElement(leftSection);
+  // showElement(timerDisplay);
   //on the timer page, show minutes, seconds, description, special color outline
 }
 
-function preventEmptyInput() {
-  if (input.value === '') {
-    error.classList.remove('hidden');
+function preventButtons() {
+  var isClicked = false;
+  for (var i = 0; i < activityButtons.length ; i++) {
+    if(!activityButtons[i].disabled) {
+      isClicked = true;
+    }
+  }
+  if(!isClicked) {
+    categoryError.classList.remove('hidden')
+  }
+}
+
+function preventAccomplish() {
+  if (!accomplishInput.value) {
+    descriptionError.classList.remove('hidden')
+  }
+}
+
+function preventMinutes() {
+  if (!minutesInput.value){
+    minutesError.classList.remove('hidden')
   }
-};
+}
+
+function preventSeconds() {
+  if (!secondsInput.value){
+    secondsError.classList.remove('hidden')
```

Comment:
We should def refactor all of these to use showElement and hideElement, otherwise looks great.